interface-definitions/containers.xml.in

```diff
@@ -16,6 +16,16 @@
               <valueless/>
             </properties>
           </leafNode>
+          <leafNode name="bind-volume">
+            <properties>
+              <help>Mount a volume into the container</help>
+              <valueHelp>
+                <format>&lt;/dir/subdir:/dir&gt;</format>
```

Comment:
This is a bad syntax.

We had that in the past with WireGuard where the endpoint was in `address:port` format, which made it very hard to validate the data.

Why not use a `<tagNode name="volume">` here with two `<leafNodes>` as `<children>` pointing to the source and destination? 


```xml
<tagNode name="volume">
  ...
  <children>
    <leafNode name="source">
      ...
    </leafNode>
    <leafNode name="desitnation">
      ...
    </leafNode>
  </children>
</tagNode>
```

Reply:
done